public/app/features/browse-dashboards/components/FolderActionsButton.tsx

```diff
@@ -86,11 +86,15 @@ export function FolderActionsButton({ folder }: Props) {
     );
   };
 
+  const managePermissions = t('browse-dashboards.folder-actions-button.manage-permissions', 'Manage permissions');
+  const move = t('browse-dashboards.folder-actions-button.move', 'Move');
+  const deleteFolderButton = t('browse-dashboards.folder-actions-button.delete', 'Delete');
```

Comment:
Maybe you could add `Label` to the end of the variable names? Otherwise it might have a different meaning, such as `deleteFolderButton` referring to an actual button.

Reply:
Thank you for suggestions, made the updates.

Reply:
Haha, I think Ash posted the same thing while I was writing it :sweat_smile: